aten/src/ATen/function_wrapper.py

```diff
@@ -146,7 +151,11 @@ def TypedDict(name, attrs, total=True):  # type: ignore
 TENSOR_METHOD_DECLARATION = CodeTemplate("""\
 ${return_type} ${api_name}(${method_formals_with_defaults}) const;
 """)
+TENSOR_METHOD_DECLARATION_UNDERSCORE = CodeTemplate("""\
```

Comment:
The variable name here would be clearer if you named it `DECLARATION_FACTORY` or something similarly descriptive about what it's actually used for, rather than `UNDERSCORE` which doesn't say anything about the intended use.